conda-recipes/llvmdev_manylinux2014/build.sh

```diff
@@ -93,3 +94,45 @@ make check-llvm-unit || exit $?
 
 # From: https://github.com/conda-forge/llvmdev-feedstock/pull/53
 make install || exit $?
+
+# run the tests, skip some on linux-32
+cd ../test
+if [[ $ARCH == 'i686' ]]; then
+    ../build/bin/llvm-lit -vv Transforms Analysis CodeGen/X86
+else
+    ../build/bin/llvm-lit -vv Transforms ExecutionEngine Analysis CodeGen/X86
+fi
+
+# Next, build compiler-rt
+
+declare -a _compiler_rt_cmake_config
+_compiler_rt_cmake_config+=(-DCMAKE_INSTALL_PREFIX:PATH=${PREFIX})
+_compiler_rt_cmake_config+=(-DCMAKE_BUILD_TYPE:STRING=Release)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_BUILTINS:BOOL=ON)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_LIBFUZZER:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_CRT:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_MEMPROF:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_PROFILE:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_SANITIZERS:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_XRAY:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_GWP_ASAN:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_BUILD_ORC:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_INCLUDE_TESTS:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DCOMPILER_RT_INCLUDE_TESTS:BOOL=OFF)
+_compiler_rt_cmake_config+=(-DLLVM_CONFIG_PATH=../../llvm/build/bin/llvm-config)
+
+cd ../../compiler-rt
+mkdir build
+cd build
+
+cmake -G'Unix Makefiles'     \
+      "${_compiler_rt_cmake_config[@]}"  \
+      ..
+
+if [ $ARCH == 'armv7l' ]; then # RPi need thread count throttling
+    make -j2 VERBOSE=1
+else
+    make -j${CPU_COUNT} VERBOSE=1
+fi
+
+make install
```

Comment:
I don't think we should be building compiler-rt separately here - do you know of an additional reason why it was built separately for the manylinux2014 build? If not, I suggest we get rid of this separate by adding all the `_compiler_rt_cmake_config` lines that set `-DCOMPILER_RT_BUILD_*` to the `_cmake_config` above, changing line 24/25 above to `cmake_config+=(-DLLVM_ENABLE_PROJECTS:STRING="lld;compiler-rt")` and removing all these lines:

```suggestion
```

Reply:
You're right, I don't see a reason of separating these. Merging into the previous `_cmake_config `, just like the llvmdev/build.sh version